.drone.yml

```diff
@@ -1001,6 +998,140 @@ volumes:
   - name: dockersock
     temp: {}
 
+---
+kind: pipeline
+type: kubernetes
+name: teleport-docker-cron-ecr
+
+trigger:
+  cron:
+    - teleport-docker-cron-ecr
+  repo:
+    include:
+      - gravitational/teleport
+
+workspace:
+  path: /go
+
+clone:
+  disable: false
+
+steps:
+  - name: Set up variables and Dockerfile
+    image: docker:git
+    environment:
+      # increment these variables when a new major/minor version is released to bump the automatic builds
+      # this only needs to be done on the master branch, as that's the branch that the Drone cron is configured for
+      # build major version images which are just teleport:x
+      CURRENT_VERSION_ROOT: v10
+      PREVIOUS_VERSION_ONE_ROOT: v9
+      PREVIOUS_VERSION_TWO_ROOT: v8
+    commands:
+      - apk --update --no-cache add curl go
+      - mkdir -p /go/build && cd /go/build
+      # CURRENT_VERSION (10)
+      - (cd /go/build.assets/tooling && go run ./cmd/query-latest $CURRENT_VERSION_ROOT > /go/build/CURRENT_VERSION_TAG.txt)
+      - echo "$(cat /go/build/CURRENT_VERSION_TAG.txt | cut -d. -f1 | tr -d '^v')" > /go/build/CURRENT_VERSION_TAG_GENERIC.txt
+      # PREVIOUS_VERSION_ONE (9)
```

Comment:
```suggestion
      # PREVIOUS_VERSION_ONE
```

Reply:
Fixed in https://github.com/gravitational/teleport/pull/14347/commits/932bcf3d957a9bb4c94634469cbcb72e79e2f8de